R/get.R

```diff
@@ -86,7 +89,12 @@ get_users <- function(src, page_size = 500, prefix = NULL, limit = Inf) {
 get_groups <- function(src, page_size = 500, prefix = NULL, limit = Inf) {
   validate_R6_class(src, "Connect")
 
-  res <- page_offset(src, src$groups(page_size = page_size, prefix = prefix), limit = limit)
+  if (!is.null(prefix)) {
+    response <- src$groups(page_size = page_size, prefix = prefix)
```

Comment:
If we don't want pagination in this case, why are we passing a `page_size` argument through here? (And to pull that thread more: why does this function (method) take a page_size argument if it doesn't do pagination?? And then, why does it exist at all?)

Reply:
When using prefix, the API only ever provides the first page of results, but the number of results on that page are still governed by `page_size`. We edited the documentation of `get_groups` to try to describe this distinction.